addons/account/tests/test_ir_actions_report.py

```diff
@@ -2,13 +2,15 @@
 import base64
 import io
 
+from PyPDF2 import PdfFileReader, PdfFileWriter
 from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+from odoo import Command
 from odoo.exceptions import UserError
-from odoo.tools import pdf
 from odoo.tests import tagged
 from odoo.tools import file_open
+from odoo.tools import pdf
```

Comment:
it's good to correct the order, but it should something like eg:

```suggestion
from PyPDF2 import PdfFileReader, PdfFileWriter

from odoo import Command
from odoo.addons.account.tests.common import AccountTestInvoicingCommon
from odoo.exceptions import UserError
from odoo.tools import pdf
from odoo.tests import tagged
from odoo.tools import file_open, pdf
```
(see [coding guidelines "Imports" section](https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports))

Reply:
done